cmd/controller/storage/postgres/postgres.go

```diff
@@ -96,6 +96,9 @@ func (iterator *tableIterator[T]) Next() bool {
 		}
 
 		iterator.iterator = storage.NewDefaultIterator[T](objects)
+		// Call next once to move the new iterator to the first element
+		iterator.iterator.Next()
+
```

Comment:
@dave-juicelabs this was pretty nasty bug - whenever we would hit the limit of the iterator (eg: 20 queued sessions) we'd crash